libraries/shared/mocks/storage_diff_repository.go

```diff
@@ -47,3 +49,17 @@ func (repository *MockStorageDiffRepository) CreateStorageDiff(rawDiff types.Raw
 	repository.CreatePassedRawDiffs = append(repository.CreatePassedRawDiffs, rawDiff)
 	return repository.CreateReturnID, repository.CreateReturnError
 }
+
+func (repository *MockStorageDiffRepository) SetCreateError(err error) {
```

Comment:
Just curious - what led you to opt for creating a setter here, as opposed to writing to the property directly?

Reply:
no idea 🤷‍♀ 😜  i'll remove that Set method in favor of just setting the property directly.